Communitter/src/main/java/com/communitter/api/service/UserService.java

```diff
@@ -62,5 +77,69 @@ public User updateUserAbout(Long id, String about) {
         return userRepo.save(existingUser);
     }
 
+    @Transactional
+    public List<UserFollowerDto> getFollowersByFollowee(Long followeeId) {
+
+        List<UserFollow> followers = userFollowRepository.findAllByFolloweeId(followeeId);
+
+        return followers.stream().map(userMapper::toUserFollowerDto).toList();
+    }
+
+    @Transactional
+    public List<UserFolloweeDto> getFolloweesByFollower(Long followerId) {
```

Comment:
With the same logic this can be changed to getFolloweesByUserId

Reply:
same here